easybuild/easyconfigs/g/gompi/gompi-2023b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'Toolchain'
+
+name = 'gompi'
+version = '2023b'
```

Comment:
Note: this was changed to `2023.09` in #18888 for now, we don't consider `2023b` to be final just yet...